ee/modules/600-flant-integration/images/flant-pricing/hooks/node_metrics.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+#
+# This is a config for node_metrics.py hook.
+configVersion: v1
+kubernetes:
+  - name: nodes
+    group: main
+    queue: /node_metrics
+    keepFullObjectsInMemory: false
+    waitForSynchronization: false
+    apiVersion: v1
+    kind: Node
+    labelSelector:
+      matchExpressions:
+        - key: "node.deckhouse.io/group"
+          operator: Exists
+    jqFilter: |
+      select(
+          .metadata.labels."node.deckhouse.io/group" != "master"
+          or
+          (
+              .spec.taints == null
+              or
+              (
+                  [
+                      .spec.taints[]
+                      | select(
+                          .key == "node-role.kubernetes.io/control-plane" or
+                          .key == "node-role.kubernetes.io/master"
+                      )
+                  ]
+                  | length == 0
+              )
+          )
+      )
+      | {
+          "nodeGroup":        .metadata.labels."node.deckhouse.io/group",
+          "pricingNodeType": (.metadata.annotations."pricing.flant.com/nodeType"       // "unknown"),
+          "virtualization":  (.metadata.annotations."node.deckhouse.io/virtualization" // "unknown")
+      }
```

Comment:
4 equal bindings look weird. It is possible to convert expressions in selects to boolean flags in output object and have only one binding for Nodes.

```
kubernetes:
  - name: nodes
    group: main
    kind: Node
    ...
    jqFilter: |
      {
          "nodeGroup":        .metadata.labels."node.deckhouse.io/group",
          "pricingNodeType": (.metadata.annotations."pricing.flant.com/nodeType"       // "unknown"),
          "virtualization":  (.metadata.annotations."node.deckhouse.io/virtualization" // "unknown"),
          "notControlPlaneNode": .metadata.labels."node.deckhouse.io/group" != "master" or ... or ... ,
          "masterNode": .metadata.labels."node.deckhouse.io/group" == "master",
          "taintedControlPlaneNode": .metadata.labels."node.deckhouse.io/group" == "master"    and ... and ... ,
          ...
      }
```

Reply:
Sounds reasonable. I'll optimize it.

Reply:
Done